docs/get_started.ipynb

```diff
@@ -555,15 +548,43 @@
       "source": [
         "That's it! You have now seen how to use TensorBoard both through the Keras callback and through `tf.summary` for more custom scenarios. "
       ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "vsowjhkBdkbK"
+      },
+      "source": [
+        "## TensorBoard.dev: Host and share your ML experiment results\n",
+        "\n",
+        "[TensorBoard.dev](https://tensorboard.dev) is a free public service that enables you to upload your TensorBoard logs and get a permalink that can be shared with everyone in academic papers, blog posts, social media, etc.  This can enable better reproducibility and collaboration.\n",
+        "\n",
+        "To use TensorBoard.dev, run the following command:\n",
+        "\n",
+        "```\n",
+        "!tensorboard dev upload \\\n",
```

Comment:
Should this be an executable code cell? To try this out, the reader would currently need to copy and paste the command to a new cell.

Reply:
done

Reply:
LGTM, thank you

Reply:
(I tested in Colab [here](https://colab.sandbox.google.com/github/tensorflow/tensorboard/blob/d6007ea9c47f4002cd1e3913bb05e6bd7de46204/docs/get_started.ipynb#scrollTo=Q3nupQL24E5E))